.github/workflows/build-test.yml

```diff
@@ -8,8 +8,22 @@ env:
     DO_NOT_TRACK: '1'
 
 on:
-    pull_request:
-        branches: ['dev', 'main', 'v2']
+  merge_group:
+  push:
+    branches: 
+      - main
+      - develop
```

Comment:
develop branch is named "dev" in this repo.

Reply:
```suggestion
      - dev
```